libs/server/src/lib/api/read-settings.ts

```diff
@@ -1,10 +1,9 @@
 import { Store } from '@nrwl/angular-console-enterprise-electron';
 import { Settings } from '../generated/graphql-types';
 
-/* tslint:disable */
 export function readSettings(store: Store): Settings {
   const settings: Settings = store.get('settings') || {};
-  if (settings.canCollectData === undefined) {
+  if ((settings as any).canCollectData === undefined) {
```

Comment:
This typecast worries me. Why is is necessary?

Reply:
The `Settings` type has `canCollectData` as required, which makes this line fail linting because it is always false.  My guess is the data model was changed at some point and this check needs to be there for old cached values.  I could disable tslint on that line instead of casting to `any` if you prefer.

Reply:
Switched to disabling tslint on that line